examples/ui.apps/src/content/jcr_root/apps/core-components-examples/clientlibs/clientlib-base-amp/styles/base/demo.less

```diff
@@ -35,4 +35,5 @@
 
 .cmp-examples-demo__component--widthFull {
     width: 100%;
+    max-width: 100%;
```

Comment:
Isn't redundant to have `width: 100%` and `max-width: 100%` on the same element?

Reply:
It is not. This case covers both cases:
- if content of demo component is wider than width of its container then it gets narrowed by `max-width: 100%`
- if content of demo component is narrower than width of its container then it gets expanded by `width: 100%`

Reply:
> `if content of demo component is wider than width of its container then it gets narrowed by max-width: 100%`

If a child element is wider than the width of its container it gets narrowed by `width:100%`. I put a small screenshot of a basic example.
<img width="877" alt="Screenshot 2022-01-14 at 17 51 58" src="https://user-images.githubusercontent.com/79856745/149545242-f7cf7982-cfd5-4661-8843-a05187db66b1.png">

Reply:
Addressed. `max-width` provided only for `--widthAuto` variant of demo component so it prevents some iframes from getting out of 100% width of component.